openpype/hosts/houdini/api/lib.py

```diff
@@ -747,3 +752,41 @@ def get_camera_from_container(container):
 
     assert len(cameras) == 1, "Camera instance must have only one camera"
     return cameras[0]
+
+
+def update_job_var_context():
+    """Validate job path to ensure it matches the settings."""
+
+    project_settings = get_current_project_settings()
+    project_settings = \
```

Comment:
This is no longer `project_settings` so maybe store it as `update_job_var_context_settings` ? Something on the lines of that